src/pages/projectSettings/Tabs.tsx

```diff
@@ -60,7 +60,9 @@ export const ProjectSettingsTabs: React.FC<Props> = ({
         >
           Save changes on page
         </Button>
-        {!useRepoSettings && <Button>Default to Repo on page</Button>}
+        {projectData && useRepoSettings && (
```

Comment:
should this be !useRepoSettings?

Reply:
No -- this was confusing me at first too, but this button actually relates to "unsetting"/nullifying all the project values on this page, not opting in to use repo settings. A user can only do so if they have already opted to use repo settings.

Reply:
If "Default to Repo on Page" is confusing, we could do something like "Set Page to Repo Defaults" ? idk if that's better